README.md

```diff
@@ -52,13 +52,15 @@ The environment in the container set up for you by this tool has many helpful fe
 
 ## Windows troubleshooting
 
-If you're on Windows, using Docker Desktop with WSL2, it is likely that the WSL2 process will chew up an enormous amount of memory on your PC. The solution is simply to:
+If you're on Windows, using Docker Desktop with WSL2, it is likely that the WSL2 process will chew up an enormous amount of memory on your PC. This is [a known issue](https://github.com/microsoft/WSL/issues/8725) with Docker Desktop with WSL2 on Windows.
+
+The solution is simply to:
 1. Shut down docker desktop
 2. Run `wsl --shutdown` from a command prompt
 3. Create a `.wslconfig` file in `%userprofile%` if it doesn't exist
 4. Add the following config to the `.wslconfig` file
 ```
 [wsl2]
-memory=6GB
+memory=8GB
 ```
-5. That's it. Save the file and restart Docker Desktop, and now WSL2 is limited to only consume a maximuim of 6GB, which should be enough.
+5. That's it. Save the file and restart Docker Desktop, and now WSL2 is limited to only consume a maximum of 8GB. It is possible to use only 6GB, but 8-10 GB is recommended and has better performance.
```

Comment:
Is this for running `psinode` or for building? If it's for building, then it likely depends on the number of cores used.

Reply:
It's the general resource limit applied to wsl2, used by the docker desktop backend.
My environment very occasionally hangs during a build, so I bumped it to 10GB and haven't yet seen any hang.

Reply:
If the limit is set based on the memory required to build (because that's the most memory intensive task), then a specific limit is probably not going to generalize very well. The amount of memory required to build scales with the number of parallel jobs, which should be dependent on the host machine and is not something that we can predict.

Reply:
Ok, how's this new wording?